addons/hr_timesheet/report/report_timesheet_templates.xml

```diff
@@ -35,7 +35,9 @@
                             <t t-set="timesheet_record_info" t-value="False"/>
                             <t t-if="show_project">
                                 <t t-set="timesheet_record_info" t-value="line.project_id.sudo().name"/>
-                                <t t-if="show_task and line.task_id" t-set="timesheet_record_info" t-value="'%s / %s' % (timesheet_record_info, line.task_id.sudo().name)"/>
+                                <t t-if="show_task and (line.task_id or line.helpdesk_ticket_id)"
+                                   t-set="timesheet_record_info"
+                                   t-value="'%s / %s' % (timesheet_record_info, (line.task_id.sudo().name if line.task_id else line.helpdesk_ticket_id.sudo().name))"/>
```

Comment:
ticket does not exist in that module.

Your fix should be in helpdesk_timesheet (enterprise repo)